### PR TITLE
Asset Management Updates: ce01issm_deploy, instrument calibrations

### PR DESCRIPTION
**Pull request by desiderr**
ce01issm-00020, nsif: dosta serial number correction 219 -> 218
instrument calibrations:
dosta-d sn 477
flort-k sn 1602
parad-k sn 20438
optaa-j sn 137, 181
spkir-j sn 252, 264, 281

All vendor calfiles to check calibrations are on alfresco. Due to file naming inconsistencies it may be easiest to locate some of them by ordering latest to earliest.

serial number change is documented in a slack channel instrument-refurb thread, oct 22 and oct 24.

### CODE REVIEW

File: calibration/FLORTK/CGINS-FLORTK-01602__20241015.csv [hunk 1]
```diff
@@ -0,0 +1,11 @@
+serial,name,value,notes
+1602,CC_dark_counts_cdom,55,date in filename comes from dev file
+1602,CC_scale_factor_cdom,9.110E-02,
+1602,CC_dark_counts_chlorophyll_a,48,
+1602,CC_scale_factor_chlorophyll_a,1.209E-02,
+1602,CC_dark_counts_volume_scatter,48,
+1602,CC_scale_factor_volume_scatter,2.984E-06,
+1602,CC_depolarization_ratio,0.039,Constant
```
Comment (aguirremar) [inline calibration/FLORTK/CGINS-FLORTK-01602__20241015.csv:8]: where can I check this info? (depolarization_ratio)

**Review by aguirremar — APPROVED** [review]

**Review by cwingard — APPROVED** [review]
Have I ever mentioned how tedious reviewing the SPKIR is? Now I am brain-mush, but I can still squash my way to the approval button....

File: calibration/FLORTK/CGINS-FLORTK-01602__20241015.csv (hunk 1 above)
Comment (desiderr) [reply calibration/FLORTK/CGINS-FLORTK-01602__20241015.csv:8]: There are 4 calibration constants in the csv calfiles that are not contained in the vendor calibration documents for (current) FLORT instruments. The constants for FLORT series D, J, and K instruments are:

angular resolution: 1.076 (this is actually a scaling factor)
scattering angle:     124 degrees
depolarization ratio: 0.039

In addition, the back scattering wavelength can be found on the corresponding vendor pdf  (Scattering Meter Calibration Sheet) below the caldate on that sheet; if it is not 700nm then the instrument needs to be sent back to the vendor for recalibration:
measurement wavelength: 700nm

All of the FLORT csv calfiles in these instrument classes contain these constants, so you can find these values in any of the FLORT-D, J, K files.

**Comment by aguirremar** [comment]
[like]  Aguirre Nunes, Mariana reacted to your message:
________________________________

**Pull request merged**